pkg/util/util.go

```diff
@@ -141,3 +125,8 @@ func Mount(source, target, fstype string, options []string) error {
 	dummyMount := mount.New("")
 	return dummyMount.Mount(source, target, fstype, options)
 }
+
+// isValidName validates Volume name
+func isValidName(name string) bool {
```

Comment:
Nit: perhaps a more descriptive name is `isValidNamingPrefix`?

Reply:
sure will fix it